docs/products/kafka/howto/kafka-tiered-storage-get-started.rst

```diff
@@ -0,0 +1,37 @@
+
+Get started with tiered storage for Aiven for Apache Kafka® 
+====================================================================
+
+Aiven for Apache Kafka®'s tiered storage expands storage beyond local disks. It stores frequently accessed data on faster tiers and less active data on cost-effective, slower tiers, ensuring both performance and cost efficiency are optimized.
```

Comment:
> It stores frequently accessed data on faster tiers

Strictly speaking, it stores the recent data on local disks, which are most likely the frequently accessed (but this is not guaranteed)

Reply:
Does rewording it like this sound better? 

Aiven for Apache Kafka®'s tiered storage optimizes resources by keeping recent data—typically the most accessed—on faster local disks. As data becomes less active, it's transferred to more economical, slower storage, balancing performance with cost efficiency.

Reply:
Sounds good!